### DISCUSSION TITLE
Documentation on mc admin config get api/heal/scanner

### DISCUSSION THREAD

**Discussion by mshanmu** [minio/minio#14826] [Q&A] [answered]
Can you please point me to documentation regarding api, heal and scanner configurations? 

The links https://github.com/minio/minio/blob/master/docs/throttle/README.md and  https://docs.min.io/docs/minio-admin-complete-guide.html#config seems to have incomplete information.

Regarding api section, I am looking to better understand and tune the cluster_deadline, list_quorum, disable_odirect variables, especially in the context of distributed cluster with HDD. 

Also, the scanner & heal pieces as well in terms of reducing the IO load on the HDDs.

Thanks for your help,

**Comment by harshavardhana** [comment, marked as answer]
https://github.com/minio/minio/tree/master/docs/config#usage-scanner

**Reply by mshanmu** [reply]
Thanks @harshavardhana  !!
